File: August 2022/August_25.java
class Solution {
    public boolean canConstruct(String ransomNote, String magazine) {
        int[] arr = new int[26];
        for(char ch : magazine.toCharArray())
            arr[ch - 'a']++;
        for(char ch : ransomNote.toCharArray())
            arr[ch - 'a']--;
        for(int i = 0; i < 26; i++)
            if(arr[i] < 0)
                return false;
        return true;
    }
}

File: August 2022/August_25.cpp
class Solution {
public:
    bool canConstruct(string ransomNote, string magazine) {
        vector<int> arr(26, 0);
        for(char &ch : magazine)
            arr[ch - 'a']++;
        for(char &ch : ransomNote)
            arr[ch - 'a']--;
        for(int i : arr)
            if(i < 0)
                return false;
        return true;
    }
};
